core/io/packet_peer.cpp

```diff
@@ -151,13 +151,13 @@ void PacketPeer::_bind_methods() {
 
 /***************/
 
-void PacketPeerStream::_set_stream_peer(REF p_peer) {
-	ERR_FAIL_COND_MSG(p_peer.is_null(), "It's not a reference to a valid Resource object.");
+void PacketPeerStream::_set_stream_peer(const Ref<StreamPeer> &p_peer) {
+	ERR_FAIL_COND_MSG(p_peer.is_null(), "It's not a reference to a valid StreamPeer object.");
 	set_stream_peer(p_peer);
 }
 
 void PacketPeerStream::_bind_methods() {
-	ClassDB::bind_method(D_METHOD("set_stream_peer", "peer"), &PacketPeerStream::set_stream_peer);
+	ClassDB::bind_method(D_METHOD("set_stream_peer", "peer"), &PacketPeerStream::_set_stream_peer);
```

Comment:
Since `_set_stream_peer` is now the same as `set_stream_peer` + an error, I'm not sure this is correct. Is there anything left using `set_stream_peer` directly? If not, should we just have `set_stream_peer` with the error? @Faless

Reply:
`set_stream_peer` was used in `3.2` by the script editor debugger, but that's no longer the case in `4.0`.
The thing that worries me about this change, is that having the error check there, would make it impossible to clear the `PacketPeerStream` so it can be reused from GDScript:
```gdscript
ppeer.set_stream_peer(_stream)
# [Do things]
# Disconnect
ppeer.set_stream_peer(null)
_stream.disconnect_from_host()
```
Of course one can create a new `ppeer`, but I still think it's weird, and I would rather remove the `_set_stream_peer` version.

Reply:
Ok, removed.